Sitemap/Voter.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Symfony\Cmf\Bundle\SeoBundle\Sitemap;
+
+/**
+ * A chain as a decorator for all tagged voter services.
+ *
+ * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ */
+class Voter extends AbstractChain implements VoterInterface
```

Comment:
So i would just fix tests and then merge?

Am 16.07.2015 um 08:55 schrieb David Buchmann:
>> @@ -0,0 +1,27 @@
>> +<?php
>> +
>> +namespace Symfony\Cmf\Bundle\SeoBundle\Sitemap;
>> +
>> +/**
>> + * A chain as a decorator for all tagged voter services.
>> + *
>> + * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
>> + */
>> +class Voter extends AbstractChain implements VoterInterface
> lets drop my idea :-/
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/symfony-cmf/SeoBundle/pull/225/files#r34760191